ui/src/main/java/de/ipb_halle/lbac/items/bean/ContainerController.java

```diff
@@ -78,30 +83,40 @@ public boolean isContainerSubComponentRendered(String typename) {
         }
         return false;
     }
-
-    public List<Integer> getTotalSlots(int dimension) {
-        if (itemBean.getContainer() == null) {
-            return new ArrayList<>();
+    
+    public List<Integer> getColumns(){
```

Comment:
NEVER initialize Iterables in getters, do it in @PostConstruct and setContainer(Container c)